exercises/concept/high-score/.docs/instructions.md

```diff
@@ -0,0 +1,91 @@
+# Instructions
+
+In this exercise, you're implementing a way to keep track of the high scores for the most popular game in your local arcade hall.
+
+## 1. Define a new high score map
+
+To make a new high score map, define the `HighScore#initialize` function which doesn't take any arguments and initializes the new Scoreboard object instance. It should take an optional argument, with the default value of an empty hash.
```

Comment:
The instructions use `HighScore` as the class name, but the example solution uses `ScoreBoard`. Also, should we be saying "methods" instead of "functions"?

Reply:
Yes, we should use the name "method" rather than "function".